docs/install.rst

```diff
@@ -106,15 +113,42 @@ package, depending on how you wish to handle virtual environments:
 
    ``/path/to/jupyter/env`` is the path to your base conda environment, and depends on the options set during your initial installation. It's probably something like ``~/conda`` or ``~/miniconda``. See the `ipython docs <https://ipython.readthedocs.io/en/stable/install/kernel_install.html>`_ for more details.
 
+   With this setup, similar to option 2, you have a single jupyter install that can run kernels from any virtual environment. The main difference is that it can run kernels from *any* virtual environment (not just conda!) but that you have to run an additional line after installing ``ipykernel``  into the environment (``python -m ipykernel install ...``).
+
    .. note:: If you're not using conda to manage your environments, the key idea is to install ``jupyter`` and ``ipywidgets`` in one environment, then install ``ipykernel`` and ``torchvision`` in the same environment as plenoptic, and then run the ``ipykernel install`` command **using the plenoptic environment's python**.
 
+The following table summarizes the advantages and disadvantages of these three choices:
+
+.. list-table::
+   :header-rows: 1
+
+   *  - Method
+      -  Advantages
+      -  Disadvantages
+   *  - 1. Everything in one environment
+      - |:white_check_mark:| Simple
+      - |:x:| Requires lots of hard drive space
+   *  - 2. ``nb_conda_kernels``
+      - |:white_check_mark:| Set up once
+      - |:x:| Initial setup more complicated
+   *  -
+      - |:white_check_mark:| Requires only one jupyter installation
+      -
+   *  -
+      - |:white_check_mark:| Automatically finds new environments with ``ipykernel`` installed
+      -
+   *  - 3. Manual kernel installation
+      - |:white_check_mark:| Flexible: works with any virtual environment setup
+      - |:x:| More complicated
+   *  -
+      - |:white_check_mark:| Requires only one jupyter installation
+      - |:x:| Extra step for each new environment
```

Comment:
At this point this now could be a standalone Medium article. 😄 Super helpful. For people that know they are going to use conda, I wonder if there is much of a payoff for option 3 (unless `nb_conda_kernels` is large, in which case there is more than just flexibility perhaps).

Reply:
`nb_conda_kernels` is pretty small.

If you're definitely using conda to manage your environments, the main reason to avoid using `nb_conda_kernels` is because you might not want jupyter to see all your environments. for example, maybe you generally don't use jupyter, so you installed it to look at notebooks in a handful of environments. and using `nb_conda_kernels` could clutter up your interface. but it would be a pretty small difference (and if the discussion makes sense to a user, I assume they're already fairly familiar with what's going on)